umm-lib/src/cmr/umm/echo10/granule.clj

```diff
@@ -57,13 +57,12 @@
                   size-unit
                   size-in-bytes
                   checksum]} data-granule
-          day-night (if day-night day-night "UNSPECIFIED")
-          size-mb (get-size-in-mb size size-unit)]
+          day-night (if day-night day-night "UNSPECIFIED")]
```

Comment:
This can be simplified to `(or day-night "UNSPECIFIED")`

Reply:
sure - not my code, but I'll make the improvement.